docs/04_instance-manager.yaml

```diff
@@ -0,0 +1,734 @@
+---
```

Comment:
All these docs that refer to EKSCF were completely removed in master branch
Also, there stills seems to be vendor files in this PR somehow

Reply:
I made some local edits in docs/04_instance-manager.yaml for `image` and must have kept it out and didn't realize you removed it.  I'll remove it.

Reply:
Are you going to make a new `instance_manager.yaml` example for the repo?

Reply:
There is already one here:
https://github.com/keikoproj/instance-manager/blob/master/config/crd/bases/instance-manager-deployment.yaml

Better to keep one copy 👍